client/src/domain/Seat/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import './style.scss';
+
+const Seat = () => {
+  const [selected, setSelected] = useState(false);
```

Comment:
I think there should also be an information if the button is available or not. If the seat is already take, it should be disabled and have a  different color.